tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/inactive/inactive_component.css

```diff
@@ -0,0 +1,78 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+.arg {
+  color: lightblue;
+  font-style: italic;
+  margin: 2px;
+}
+
+.code {
+  font-family: Roboto Mono;
+  margin: 10px;
+}
+
+.container {
+  height: 100%;
+  font-family: Roboto;
+  font-size: 15px;
+  padding: 50px;
+}
+
+.details-container {
+  vertical-align: middle;
+}
+
+.details {
+  align-content: left;
```

Comment:
`align-content` does not do anything on non-flexbox and non-grid context. Extraneous?

Reply:
Removed.